contracts/staking/IStaking.sol

```diff
@@ -176,39 +170,11 @@ interface IStaking {
 
     /// @notice Reduces the liquid T stake amount by the provided amount and
     ///         withdraws T to the owner. Reverts if there is at least one
-    ///         authorization higher than the sum of the legacy stake and
-    ///         remaining liquid T stake or if the unstake amount is higher than
-    ///         the liquid T stake amount. Can be called only by the delegation
-    ///         owner or the staking provider.
-    function unstakeT(address stakingProvider, uint96 amount) external;
-
-    /// @notice Sets the legacy KEEP staking contract active stake amount cached
-    ///         in T staking contract to 0. Reverts if the amount of liquid T
-    ///         staked in T staking contract is lower than the highest
-    ///         application authorization. This function allows to unstake from
-    ///         KEEP staking contract and still being able to operate in T
-    ///         network and earning rewards based on the liquid T staked. Can be
-    ///         called only by the delegation owner or the staking provider.
-    function unstakeKeep(address stakingProvider) external;
-
-    /// @notice Sets to 0 the amount of T that is cached from the legacy
-    ///         NU staking contract. Reverts if there is at least one
-    ///         authorization higher than the sum of remaining legacy NU stake
-    ///         and native T stake for that staking provider or if the unstaked
-    ///         amount is higher than the cached legacy stake amount. If succeeded,
-    ///         the legacy NU stake can be partially or fully undelegated on
-    ///         the legacy NU staking contract. This function allows to unstake
-    ///         from NU staking contract while still being able to operate in
-    ///         T network and earning rewards based on the native T staked.
-    ///         Can be called only by the stake owner or the staking provider.
-    function unstakeNu(address stakingProvider) external;
-
-    /// @notice Sets cached legacy stake amount to 0, sets the liquid T stake
-    ///         amount to 0 and withdraws all liquid T from the stake to the
-    ///         owner. Reverts if there is at least one non-zero authorization.
+    ///         authorization higher than the remaining liquid T stake or
+    ///         if the unstake amount is higher than the liquid T stake amount.
```

Comment:
I think we can change "`liquid T stake`" (which doesn't make much sense now) into just "`T stake`" everywhere

Reply:
✔️